lib/gui/scss/modules/_bootstrap.scss

```diff
@@ -36,7 +36,16 @@ body {
   cursor: default;
 }
 
-.tooltip {
+.tooltip,
+.popover {
   word-wrap: break-word;
 }
 
+.page-main .popover-title {
```

Comment:
We shouldn't add page specific styles in the `_bootstrap.scss` file, which is meant to hold Bootstrap specific styles.

By seeing this, I think the `.page-main` prefixes are unnecessary (the styles apply to the popovers independently of the page), so just dropping `.page-main` is probably enough.